phpBB/includes/functions_posting.php

```diff
@@ -52,9 +52,28 @@ function generate_smilies($mode, $forum_id)
 
 		page_header($user->lang['SMILIES']);
 
-		$sql = 'SELECT COUNT(smiley_id) AS item_count
-			FROM ' . SMILIES_TABLE . '
-			GROUP BY smiley_url';
+		$sql_ary = [
```

Comment:
So you're building this array but you're not actually using but instead run the sql query with `$sql` which ends up being undefined? I think you forgot to add a new line of code. ;)

Reply:
Thanks, fixed.